en/docs/getting-started/overview.md

```diff
@@ -1,99 +1,101 @@
 # Overview
 
-WSO2 API Manager is a fully open-source solution for end to end API Management in the cloud, on-prem or in hybrid environments. It comes with an [Apache Software License Version 2.0](http://www.apache.org/licenses/LICENSE-2.0) which makes it **free to use**. It allows API developers to design, publish, and manage the lifecycle of APIs and API product managers to create API products from one or more APIs. It hosts an application developer portal which helps in building and managing a developer community for your APIs. Its cloud-native API gateway is used for securing, routing, controlling and monitoring your API traffic in a scalable manner.
+<div style="width: 100%; overflow: hidden;">
+  
+<div class="rightContentOverview" style="width:40% !important;"> WSO2 API Manager is a fully open-source API management platform. It supports API designing, API publishing,  lifecycle management, application development, API security,  rate limiting, viewing statistics of APIs, as well as connecting APIs, API Products, and endpoints.
```

Comment:
We need to add a bit of space between the text and the video.
<img width="810" alt="Screenshot 2020-12-09 at 17 03 35" src="https://user-images.githubusercontent.com/5195851/101628531-fa371380-3a45-11eb-9bf7-f06f7c6783dd.png">